src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SentReferralSummariesDTO.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ReferralSummary
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class SentReferralSummariesDTO(
+  val id: UUID,
+  val sentAt: OffsetDateTime,
+  val sentBy: AuthUserDTO,
+  val referenceNumber: String,
+  val assignedTo: AuthUserDTO?,
+  val serviceUser: ServiceUserDTO?,
+  val serviceProvider: ServiceProviderDTO?,
```

Comment:
same with this

Reply:
I was thinking about unassigned cases and thought it will not have serviceUser or serviceProvider?

Reply:
service provider is the organization that runs the intervention. so again every referral will have an associated service provider.